src/m_menu.c

```diff
@@ -1467,9 +1475,13 @@ enum
 menuitem_t MouseMenu[]=
 {
   // [FG] alternative text for missing menu graphics lumps
-  {2,"M_HORSEN",M_MouseHoriz,'h', "HORIZONTAL"},
+  {2,"M_HORSEN",M_MouseHoriz,'h', "HORIZONTAL: TURN"},
+  {-1,"",0},
+  {2,"M_HORSEN2",M_MouseHoriz2,'h', "HORIZONTAL: STRAFE"},
```

Comment:
`M_HORSEN2` is 9 characters long, so it is truncated to `M_HORSEN`.

Reply:
LOL! What do you suggest?

Reply:
Alternatively, we can move the mouse sensitivity to the General menu and keep the vanilla look of the Mouse Sensitivity menu. We can use the mini thermo in the General menu.

Reply:
He, but we have a "Mouse Sensitivity" menu for this exact reason. I'd be surprised to find one specific mouse sensitivity setting elsewhere. Also, even if we move just one vertical mouse sensitivity setting out of the way, we'd still end up with two "Horizontal" settings and thus two `M_HORSEN` patches.

I am inclined to do just this and say "goodbye" to patches drawing in the "Mouse Sensitivity" menu:

```patch
--- a/src/m_menu.c
+++ b/src/m_menu.c
@@ -1475,13 +1475,13 @@ enum
 menuitem_t MouseMenu[]=
 {
   // [FG] alternative text for missing menu graphics lumps
-  {2,"M_HORSEN",M_MouseHoriz,'h', "HORIZONTAL: TURN"},
+  {2,"",M_MouseHoriz,'h', "HORIZONTAL: TURN"},
   {-1,"",0},
-  {2,"M_HORSEN2",M_MouseHoriz2,'h', "HORIZONTAL: STRAFE"},
+  {2,"",M_MouseHoriz2,'h', "HORIZONTAL: STRAFE"},
   {-1,"",0},
-  {2,"M_VERSEN",M_MouseVert,'v', "VERTICAL: MOVE"},
+  {2,"",M_MouseVert,'v', "VERTICAL: MOVE"},
   {-1,"",0},
-  {2,"M_VERSEN2",M_MouseVert2,'v', "VERTICAL: LOOK"},
+  {2,"",M_MouseVert2,'v', "VERTICAL: LOOK"},
   {-1,"",0}
 };

```

Reply:
Sure, I agree with Crispy's approach.